test/parallel/test-console.js

```diff
@@ -138,6 +138,16 @@ console.timeEnd();
 console.time(NaN);
 console.timeEnd(NaN);
 
+// make sure calling time twice without timeEnd doesn't reset the timer.
+console.time('test');
+const time = console._times.get('test');
+setTimeout(() => {
```

Comment:
Can we get rid of the timeout?

Reply:
ita just to make sure we have a noticeable difference if something goes wrong, I'm tired of flakey tests from too small timeouts etc